tests/e2e/pact/package.json

```diff
@@ -33,12 +30,15 @@
     "@types/supertest": "2.0.9",
     "@typescript-eslint/eslint-plugin": "3.2.0",
     "@typescript-eslint/parser": "3.2.0",
+    "@hapi/boom": "^9.1.0",
```

Comment:
why are these moving down :sob:

Reply:
@h is before @t in alphabet

Reply:
Love that you just tagged two random developers in this PR :joy:

Reply:
lol oops